README.md

```diff
@@ -37,23 +116,32 @@ repository, after your initial bazel build:
 
     bazel-bin/external/io_bazel_rules_go_repository_tools/bin/gazelle
 
-_Note_: If you cant find the gazelle binary in the path mentioned above, try to
-update the mlocate database (`sudo updatedb` or the equivalent in macOS) and
-run `locate gazelle`. The gazelle binary should typically be in
+_Note_: If you cant find the gazelle binary in the path mentioned above,
+try to update the mlocate database and run `locate gazelle`. The gazelle
+binary should typically be in
 
     $HOME/.cache/bazel/_bazel_<username>/<somelongfoldername>/external/io_bazel_rules_go_repository_tools/bin/gazelle
 
 ## Test environment ##
 
-Manager tests require access to a Kubernetes cluster. We recommend Kubernetes
-version >=1.5.2 due to its improved support for Third-Party Resources. Each
+Manager tests require access to a Kubernetes v1.5.2 cluster. Each
```

Comment:
It's very likely we'll be using latest Kubernetes to get better support for extensibility, and we should start testing 1.6 soon.